src/backend/api/handlers/decorators.py

```diff
@@ -0,0 +1,39 @@
+from flask import request
+from functools import wraps
+
+from backend.common.models.team import Team
+from backend.common.queries.exceptions import DoesNotExistException
+
+
+def api_authenticated(func):
```

Comment:
We should be able to type these decorators by using `Callable` https://docs.python.org/3/library/typing.html#typing.Callable - I'm not super sure about how the args/kwargs works for typing though

Reply:
At minimum we should be able to add none return types or something right?